hudi-common/src/main/java/org/apache/hudi/common/model/WriteOperationType.java

```diff
@@ -48,6 +48,8 @@
   INSERT_OVERWRITE_TABLE("insert_overwrite_table"),
   // compact
   COMPACT("compact"),
+  // alter schema
+  ALTER_SCHEMA("alter_schema"),
   // used for old version
```

Comment:
I'm not sure if `ALTER SCHEMA` belong to the write operation, and if the ddl operation should be put in `BaseHoodieWriteClient`.

Reply:
Thanks for your review.
I think it should belong to write opertion.  DDL operations of sparksql will treat it as a write operation.
of course， if  you think this is inappropriate， i will remove it from WriteOperation

Reply:
Makes sense